app/src/main/java/com/example/concordia_campus_guide/LocationFragment/LocationFragmentViewModel.java

```diff
@@ -88,7 +88,8 @@ private void addMarker(GoogleMap map, LatLng centerPos, BuildingCode buildingCod
                         .flat(true)
                         .anchor(0.5f,0.5f)
                         .alpha(0.90f)
-                        .title(buildingCode.toString())
+                        //This line should be included whenever we test the UI for the marker:
+                        //.title(buildingCode.toString())
```

Comment:
Is there really no other way to make the ui test without having to uncomment this line everytime ? If in the future we require to have similar behavior to test classes it might be a pain to uncomment all those future lines

Reply:
I agree that this will be confusing in the future. Maybe mocking that data for the test would work?

Reply:
It might be possible. However, at the moment, I didn't find another solution